src/http/server/response.cr

```diff
@@ -113,17 +123,18 @@ class HTTP::Server
     # Generates an error response using *message* and *code*.
     #
     # Calls `reset` and then writes the given message.
-    def respond_with_error(message = "Internal Server Error", code = 500)
+    def respond_with_error(status : HTTP::Status = HTTP::Status::INTERNAL_SERVER_ERROR, status_message = nil)
```

Comment:
Note that this is a breaking change. We should either postpone merging this PR until 0.28.0 or else preserve the old overload.